### ISSUE TITLE
Jaws 2019 doesn't read fieldset legend with Edge

### ISSUE THREAD

**Issue by juliemoynat-tanaguru** [FreedomScientific/VFO-standards-support#141]
### Summary

Jaws doesn't read neither the `legend` of a `fieldset` nor the label (`aria-labelledby` or `aria-label`) of a `role="group"` with Microsoft Edge (it's OK with Internet Explorer 11).

(I don't know if it's a Jaws or Edge bug.)

### Expected result

I expect Jaws reading the legend when I focus the field.

### Actual result

When I focus a field, Jaws doesn't read the legend.

### Example

Code for tests:
https://codepen.io/juliemoynat/pen/BGPyoz

#### JAWS version and build number
JAWS 2019.1810.64 - October 2018

 
#### Operating System and version
Windows 10

#### Browser and version:
Microsoft Edge 44.17763.1.0

**Comment by stevefaulkner** [comment]
Given that Edge in its current state [will not continue to be developed](https://twitter.com/stevefaulkner/status/1070761434450141185) closing this as won't fix
